src/test/setup.ts

```diff
@@ -81,6 +82,17 @@ let scrollBy: jest.SpyInstance;
 let mockGa: any;
 
 resetModules();
+
+beforeAll(async() => {
+  // Add DOMParser if it doesn't exists since it is require by archiveTreeUtils
+  if (!(global as any).DOMParser) {
+    (global as any).DOMParser = new JSDOM().window.DOMParser;
```

Comment:
doesn't jest add this already?

Reply:
it started to fail for `test:browser` with `ReferenceError: DOMParser is not defined` after adding `disableArchiveTreeCaching`. I guess this is because of importing the `archiveTreeUtils` after resetting modules
I'll try to find another workaround